static/sass/_patterns_flex-grid.scss

```diff
@@ -0,0 +1,48 @@
+@mixin p-flex-grid {
```

Comment:
Nice pattern. I have a concern about the name. We should name patterns by the purpose and not the Styling they are applying. For example, in the future we may swap flex for grid or something new. Then the name of the pattern will become confusing.

Reply:
`flex` is kind of meant to be coming from 'flexibility' rather then 'flexbox' - meaning flexible grid, or adaptive grid. But any name suggestions welcome.

Reply:
I understand but you can see how it could become confusing. `p-agile-grid`, `p-adaptive-grid`, `p-fliud-grid`?